site/content/en/docs/tutorials/kubevirt.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "KubeVirt support"
```

Comment:
I would change this to a 
How To... format, for example how to use KubeVirt with minikube

Reply:
this has been updated! and the PR is ready for a rereview!